test_fixtures/extract_strings/f_option/input_files/app.go

```diff
@@ -39,8 +39,9 @@ var appHelpTemplate = `{{.Title "NAME:"}}
 
 func NewApp(cmdRunner command_runner.Runner, metadatas ...command_metadata.CommandMetadata) (app *cli.App) {
 	helpCommand := cli.Command{
-		Name:        "help",
-		ShortName:   "h",
+		Name:      "help",
+		ShortName: "h",
+		//locales:fr
```

Comment:
You added a comment here. Is this intentional?

Reply:
Yeah, this is why there's a change in app.go.extracted.json for this file! This package doesn't have traditional unit tests, it's got these end-to-end test fixtures instead.